mergify_engine/worker.py

```diff
@@ -149,7 +152,7 @@ async def push(
         gh_pull=pull_number,
         event_type=event_type,
     )
-    return (ret[0], payload)
+    return (T_MessageID(ret[0]), payload)
```

Comment:
I'd rather do:
```suggestion
    return (typing.cast(T_MessageID, ret[0]), payload)
```
no?
I'm surprised this is not making mypy complain.

Reply:
I guess it works because aredis return Any, not sure

Reply:
According the doc, when you use `NewType` it's equivalent at mypy time as :

```
class T_MessageID(str):
    def __init__(self, x: str) -> None:
         pass
```
and at runtime to:
```
def T_MessageID(self, x: str) -> str:
    return x
```

In our case redis return `Any`, so it can't check anything and just trust us.

Reply:
I just push a third syntax to do the same for thing returning Any ;)